tests/components/shelly/conftest.py

```diff
@@ -190,3 +278,86 @@ async def rpc_wrapper(hass):
     wrapper.async_setup()
 
     return wrapper
+
+
+@pytest.fixture
+async def rpc_poll_wrapper(hass):
```

Comment:
What is this used for? currently we only use the RPC polling wrapper for specific sensors from the `sensor` platform which needs polling, sensors migrated from the `binary_sensor` does not use polling and you don't use it in your code either.
https://github.com/home-assistant/core/blob/83b426deb0e14689d71943102b12f9086bdf2073/homeassistant/components/shelly/sensor.py#L389

Reply:
I have removed the rpc_poll_wrapper and simply added the RpcPollingWrapper setup in the rpc_wrapper fixture.